src/bindings/jna/libelektra4j/src/test/java/org/libelektra/README.md

```diff
@@ -0,0 +1,22 @@
+Unit tests can be run after importing jUnit, JNA and the libelektra4j java library into a project (eclipse, netbeans, intelliJ, ...). Alternatively, you can use maven to both build the project or just run the unit tests.
```

Comment:
No, it was like that when i started working on it so i left it in there for now, but the "outermost" readme in the jna directory already contains test instructions. So I'd drop those here.